pkg/controller/elasticsearch/driver/upgrade_deletion_driver.go

```diff
@@ -0,0 +1,154 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I think these file names reflect a previous version of this code. You removed the driver